volumes/miovision/sql/views/create-view-volumes_daily.sql

```diff
@@ -8,10 +8,16 @@ CREATE VIEW miovision_api.volumes_daily AS (
         v.holiday,
         v.datetime_bins_missing,
         v.unacceptable_gap_minutes,
-        v.avg_historical_gap_vol
+        v.avg_historical_gap_vol,
+        array_agg(ar.notes ORDER BY ar.range_start, ar.uid)
```

Comment:
seems like if there are no notes the value would look like `{NULL}`, should we make it actually null with 
`array_agg(ar.notes ORDER BY ar.range_start, ar.uid) filter(where notes is not null) ` instead?

Reply:
Good suggestion! Also added column comments for those columns.